.github/pull_request_template.md

```diff
@@ -1,23 +1,12 @@
-<!--- Provide a general summary of your changes in the Title above -->
 
-## Ticket
-<!--- 
-Add a reference to the ticket this PR relates to, if any. 
-You can write 'Fixes #100' to auto close ticket #100 on merge.
-Or write 'Re #100' to reference ticket #100 without changing its status.
--->
+<!--- Add a reference to the Github issue this PR relates to, if any, eg: 'Fixes #100' -->
+Fixes #
```

Comment:
In line with our Commit Message / Title rules, I'd change this to "fix", which also works
```suggestion
Fix #
```